### PR TITLE
Fix for the Menu presets

### PR DESCRIPTION
**Pull request by hacki65**
Pull Request for Issue # .
#18905 
### Summary of Changes
Reordered menu entries and added seperators (like in Joomla! 3.7). Fix the icon for "Menus".

### CODE REVIEW

**Comment by infograf768** [comment]
Similar to https://github.com/joomla/joomla-cms/pull/18908 ?

**Comment by Quy** [comment]
The other PR is for the Contents menu. This PR is for the Menus menu.

**Comment by hacki65** [comment]
@infograf768 Yes, similar to #18908, but this fixes more issues in the whole admin menu. Since 3.8 Joomla! has presets for the admin menu and can be changed in the admin menu module.

In order that both presets give same result i have changed both presets. It gives the same ordering (incl. the seperators) for admin menu as it was in J3.7.

In preset "modern.xml" the only change is that the menu entry "Extensions" would be "Manage" with an alternative menu struktur.

Also it fixes the problem with the menu entry "Menus" where the icons for language or home has no space between the string and the icon.

If this PR will be merged, than the other could be closed. But nevertheless thanks to @alikon for his PR.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 46a34a8c2947133e2df751113d5eb8b954606146<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18918">issues.joomla.org/tracker/joomla-cms/18918</a>.</sub>

**Comment by ladyjer** [comment]
I have tested this item :white_check_mark: successfully on 46a34a8c2947133e2df751113d5eb8b954606146<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18918">issues.joomla.org/tracker/joomla-cms/18918</a>.</sub>

**Comment by infograf768** [comment]
@izharaazmi 
Please check and confirm.

**Comment by izharaazmi** [comment]
I have tested this item :white_check_mark: successfully on 46a34a8c2947133e2df751113d5eb8b954606146<br /><br />Looks better now :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18918">issues.joomla.org/tracker/joomla-cms/18918</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after three successful tests.

**Comment by hacki65** [comment]
Thanks for testing, guys.

File: administrator/components/com_menus/presets/joomla.xml [hunk 1]
```diff
@@ -213,7 +216,7 @@
 			sql_group="a.id, m.language, l.image"
 			sql_order="a.title ASC">
 			<menuitem
-				title="{sql:title}"
+				title="{sql:title} "
```
Comment (alikon) [inline administrator/components/com_menus/presets/joomla.xml:219]: why the extra space `title="{sql:title} "`  ?

**Comment by izharaazmi** [comment]
Space is to add a space between title and icon.

**Comment by hacki65** [comment]
@alikon Yep, a simple, but effective solution. With the original presets the icon for home or language image has no space - in order it is place direct behind the title.

**Pull request merged**